assets/js/editor-components/color-panel/index.tsx

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/**
+ * External dependencies
+ */
+import { useMemo } from '@wordpress/element';
+import {
+	store as blockEditorStore,
+	getColorClassName,
+	InspectorControls,
+	useBlockEditContext,
+	__experimentalColorGradientSettingsDropdown as ColorGradientSettingsDropdown,
+	__experimentalUseMultipleOriginColorsAndGradients as useMultipleOriginColorsAndGradients,
```

Comment:
We should disable the `@wordpress/no-unsafe-wp-apis` ESLint rule in these lines.